bolt.yaml

```diff
@@ -0,0 +1,20 @@
+# puppetdb:
+#   server_urls: ["https://foo.example.com:8081"]
+#   cacert: /path/to/ca.pem
+#   token: ~/.puppetlabs/token
+# pcp:
+#   service-url: "https://foo.example.com:8143"
+#   cacert: /path/to/ca.pem
+#   task-environment: production
```

Comment:
We should probably omit `task-environment`, since that's quite an advanced use case